chia/rpc/wallet_rpc_api.py

```diff
@@ -2441,8 +2439,9 @@ async def nft_set_did_bulk(self, request):
 
             for tx in refined_tx_list:
                 await self.service.wallet_state_manager.add_pending_transaction(tx)
-            for coin in coin_ids:
-                await nft_wallet.update_coin_status(coin, True)
+            for coin_id in coin_ids:
+                # Ignore update_coin_status is not included in WalletProtocol
```

Comment:
I think this, and similar below, could be avoided with a `nft_wallet: NFTWallet` near the top and then using `WalletStateManager.get_wallet()`.  Other than the uncertainty around mypy not complaining that this may not even be assigned...

Reply:
```suggestion
```